### PR TITLE
Update it.json

### PR DESCRIPTION
**Pull request by enricomg**
patch

### CODE REVIEW

File: js/web/_i18n/it.json [hunk 1]
```diff
@@ -233,13 +233,13 @@
 		"Boxes.Technologies.NoTechs": "Hai raggiunto la fine di questa era",
 		"Boxes.Technologies.Resource": "Risorsa",
 		"Boxes.Technologies.Title": "Costo della ricerca per",
-		"Boxes.Units.AlcaHarvest": "Your Alcatraz is ready to harvest!", //Todo: Translate
+		"Boxes.Units.AlcaHarvest": "Il tuo Alcatraz è pronto per essere raccolto!", 
 		"Boxes.Units.Attack": "Attacco",
 		"Boxes.Units.Bind": "Caserma",
 		"Boxes.Units.Defend": "Difesa",
-		"Boxes.Units.NextUnitsIn": "Le prossime unità __count__ arriveranno in <span class = \"alca-countdown \"> </span> alle ore __harvest__",
+		"Boxes.Units.NextUnitsIn": "Le prossime unità __contare__ arriveranno in <span class = \"alca-conto alla rovescia \"> </span> alle ore __raccogliere__",
```
Comment (Th3C0D3R) [inline js/web/_i18n/it.json:240]: Dont translate words with __ befor and after it

**Comment by Th3C0D3R** [comment]
Please fix the translation (dont translate words with __ before and after) and please remove the //Todo: Translate if the line is translated

File: js/web/_i18n/it.json [hunk 2]
```diff
@@ -233,13 +233,13 @@
 		"Boxes.Technologies.NoTechs": "Hai raggiunto la fine di questa era",
 		"Boxes.Technologies.Resource": "Risorsa",
 		"Boxes.Technologies.Title": "Costo della ricerca per",
-		"Boxes.Units.AlcaHarvest": "Your Alcatraz is ready to harvest!", //Todo: Translate
+		"Boxes.Units.AlcaHarvest": "Il tuo Alcatraz è pronto per essere raccolto!", 
 		"Boxes.Units.Attack": "Attacco",
 		"Boxes.Units.Bind": "Caserma",
 		"Boxes.Units.Defend": "Difesa",
-		"Boxes.Units.NextUnitsIn": "Le prossime unità __count__ arriveranno in <span class = \"alca-countdown \"> </span> alle ore __harvest__",
+		"Boxes.Units.NextUnitsIn": "Le prossime unità __count__ arriveranno in <span class = \"alca-conto alla rovescia \"> </span> alle ore __raccogliere__",
```
Comment (Th3C0D3R) [inline js/web/_i18n/it.json:240]: Still translated word with __ 
At the end of the line is one

**Comment by Th3C0D3R** [comment]
Still not right, at the end of the line where you translated __count__ you translated also a other word

File: js/web/_i18n/it.json (hunk 2 above)
Comment (andrgin) [reply js/web/_i18n/it.json:240]: Also dont translate the HTML classes (alca-countdown).

**Pull request merged**